### DISCUSSION TITLE
ChatBot

### DISCUSSION THREAD

**Discussion by eros1006** [DedicatedSoftwareDev/Lending-Bot#14] [Announcements]
How to create chatbot and connect to telegram?

**Comment by websparkledev** [comment]
Telegram bot is designed to have a conversation with end-users and give unique solutions.

**Comment by VeteranSoftDev** [comment]
ChatBot should be integrated to ChatGPT and LLM module so that it can generate answers according to proper prompt.
Open AI integration is crucial to this project.
